pims/base_frames.py

```diff
@@ -154,6 +160,10 @@ def __getitem__(self, key):
     def __iter__(self):
         return iter(self[:])
 
+    @property
+    def shape(self):
+        return (len(self), *self.frame_shape)
```

Comment:
Probably can just change this to `(len(self),) + self.frame_shape`.  Though @caspervdw has said as much.

Reply:
This may break for cases where we do not know their length, but in that case `len(obj)` is already broken so I am not super worried.